openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -61,6 +61,7 @@ class LearningContext(TimeStampedModel):
     title = models.CharField(max_length=255)
     published_at = models.DateTimeField(null=False)
     published_version = models.CharField(max_length=255)
+    entrance_exam_id = models.CharField(max_length=255, null=True)
```

Comment:
This field will go in to `CourseContext` since all learning contexts will not have exams.